main/src/ca/uwaterloo/flix/api/lsp/provider/completion/KeywordCompleter.scala

```diff
@@ -63,61 +66,105 @@ object KeywordCompleter {
   def getExprKeywords: Iterable[Completion] =
     List(
       // A
-      Completion.KeywordCompletion("and"      , Priority.High),
-      Completion.KeywordCompletion("as"       , Priority.Low),
+      Completion.KeywordCompletion("and"            , Priority.Higher),
+      Completion.KeywordCollectionCompletion("Array", Priority.Low),
+      Completion.KeywordCompletion("as"             , Priority.High),
+      // C
+      Completion.KeywordCompletion("catch"            , Priority.High),
+      Completion.KeywordCallCompletion("checked_cast" , Priority.Higher),
+      Completion.KeywordCallCompletion("checked_ecast", Priority.Low),
+      Completion.KeywordCompletion("choose"           , Priority.Lowest),
+      Completion.KeywordCompletion("choose*"          , Priority.Lowest),
       // D
-      Completion.KeywordCompletion("def"      , Priority.Higher),
-      Completion.KeywordCompletion("discard"  , Priority.Low),
-      Completion.KeywordCompletion("do"       , Priority.High),
+      Completion.KeywordCallCompletion("debug!!", Priority.Lowest),
+      Completion.KeywordCallCompletion("debug!" , Priority.Lowest),
+      Completion.KeywordCallCompletion("debug"  , Priority.Lower),
```

Comment:
should the longer debugs not come after?

Reply:
I thought so, but my sorting them alphabetically in my shell actually gave this order. But I can change it if it feels odd

Reply:
Dump debug*